src/components/Checkbox/Checkbox.tsx

```diff
@@ -34,6 +34,12 @@ export type CheckboxProps = Omit<CheckboxInputProps, 'id'> & {
  * Checkbox control indicating if something is selected or unselected.
  *
  * Requires either a visible label or an accessible name.
+ *
+ * NOTE: usually, we would re-export subcomponents with
+ *   Checkbox.Input = CheckboxInput;
+ *   Checkbox.Label = CheckboxLabel;
+ * but this does not compile with Typescript since Checkbox itself is a
+ * forwarded ref component. Thus Input and Label must be imported separately.
```

Comment:
😭 probably could figure out this type issue but I remember it being a pain in the past

Reply:
Nothing a little `@ts-ignore` couldn't solve (probably)

Reply:
I tried `@ts-ignore` and `@ts-expect-error` and restarting VSCode but sadly no dice :(
![Screen Shot 2022-04-22 at 1 09 12 PM](https://user-images.githubusercontent.com/15840841/164762055-5ce50ce2-c728-4536-9ac4-8b758326061b.png)

down to pair on it at some point though

Reply:
Okay, let's pair on it sometime.

Looks like the ts-ignore lets us assign to the Checkbox function, but TS still doesn't know about what we added when we try to use them.

Probably we need a type annotation like `const Checkbox: SuperAmazingForwardRef = React.forwardRef` to tell TS about it.